students/M_Sunday/lesson09/Mailroom_Part_5.py

```diff
@@ -0,0 +1,173 @@
+#!/usr/bin/python
+from operator import itemgetter
+
+# Lesson 09, Mailroom Part 5 Assignment
+
+
+class UserInputs(object):
+
+    @staticmethod
+    def get_menu_option():
+        sel_ = input("------> "
+                     "Select an option from the menu: ")
+        return sel_
```

Comment:
using underscores at the end of variable names is _not_ a common patter in python